docs/ISuite/50-Development/using-camel-simple-expression-language-4688083.md

```diff
@@ -256,7 +256,8 @@ Incoming message body
 </td>
 <td valign="top">
 
-Incoming message body
+Incoming message body<br/>
+ <i>Deprecated from Camel version 3.12.x</i>
```

Comment:
```suggestion
 <i>This expression is deprecated from Apache Camel version 3.12.x.</i>
```

Reply:
Thank you a lot for this feedback! The documentation will be changed accordingly.